parktain/main.py

```diff
@@ -4,10 +4,18 @@
 from datetime import datetime
 from os.path import abspath, dirname, join
 import re
+from datetime import datetime
+from urllib.parse import urlparse
 
 # 3rd party library
 from gendo import Gendo
 
+# sqlalchemey stuff
```

Comment:
Move the sqlalchemy imports under the 3rd-party section. Add a new section with `# Local libaray` and import the models. http://pep8.org/#imports